UaClient.UnitTests/UnitTests/UaApplicationOptionsTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Workstation.ServiceModel.Ua;
+using Xunit;
+
+namespace Workstation.UaClient.UnitTests
+{
+    public class UaApplicationOptionsTests
+    {
+        [Fact]
+        public void UaTcpTransportChannelOptionsDefaults()
+        {
+            var lowestBufferSize = 1024u;
```

Comment:
In the spec, the smallest buffer size is 8192. 
The buffer holds a chunk.  One or more chunks form a message.
You can limit the number of chunks and the size of a message, but you can also disable these two limits by setting them to zero.
So please make: 
lowestBufferSize = 8192u; 
lowestMessageSize = 0u;
lowestChunkCount = 0u;

Reply:
I see, increasing the `lowestBufferSize` to 8192u makes totally sense. That an unsigned integer is 0 or greater, is obvious, we do not need to test that. On the other hand, we are only testing the default values here. At first I thought I simply test against the exact values. Shall I remove the tests for `MaxMessageSize` and `MaxChunkCount` then?

Reply:
Yes, please.

Reply:
Done.